### ISSUE TITLE
Cannot select concepts when alphabet too large

### ISSUE THREAD

**Issue by Kordishal** [NatLibFi/Skosmos#718]
## At which URL did you encounter the problem?
http://bartoc-skosmos.unibas.ch/UDC/en/index?clang=zh
## What steps will reproduce the problem?
1. Try to scroll down and select a concept.

## What is the expected output? What do you see instead?
It will jump back up, before reaching the bottom of the side bar. Behavior needs to be changed to prevent 
jumping back up or a different way to display the Chinese alphabet.

## What browser did you use? (eg. Firefox, Chrome, Safari, Internet explorer)
Firefox, Desktop.

**Comment by osma** [comment]
Right. I'm pretty sure forming an alphabetical index for Chinese terms by just taking the first characters of each term is not the right way, though I'm unsure what the correct way would be.

In any case, the scrolling should work but it doesn't.
